AdminClient/src/app/products/products.component.ts

```diff
@@ -40,31 +41,62 @@ export class ProductsComponent implements AfterViewInit, OnInit {
     this.table.dataSource = this.dataSource;
   }
 
-  handleReadDetails(id: number) {
-    this.router.navigate(['./', 'details', id]);
+  handleReadDetails(product: Product) {
+    const dialogRef = this.dialog.open(ProductDetailsDialogComponent, {
+      width: '500px',
```

Comment:
Prefer relevant units. Here you can use %, vh, vw, em etc.

Reply:
I'm now setting the width relatively but with an absolute maximum - to prevent it from being too large